recipes/newrelic/infrastructure/amazonlinux.yml

```diff
@@ -107,6 +115,17 @@ install:
             echo "touch is required to run the newrelic install. Please install touch and re-run the installation." >&2
             exit 15
           fi
+        - |
+          IS_INFRA_AVAILABLE=$(curl -Is {{.NEW_RELIC_DOWNLOAD_URL}}infrastructure_agent/linux/yum/amazonlinux/{{.AMAZON_LINUX_VERSION}}/{{.ARCH}}/newrelic-infra.repo | grep " 2[0-9][0-9] " | wc -l)
+          if [ $IS_INFRA_AVAILABLE -eq 0 ] ; then
```

Comment:
What does this grep `2[0-9][0-9]` do? ☝️

Reply:
We curl just headers and look for the 2xx response code.

Reply:
Oh, I didn't get the headers detail 🤦